### DISCUSSION TITLE
Dexcom G6: "Temporary Sensor Issue" / "Calibration Error 1"

### DISCUSSION THREAD

**Discussion by loopy321** [JohanDegraeve/xdripswift#374] [General]
I started a sensor, got two readings and calibrated.  Now I see "temporary sensor issue" on the Bluetooth screen.  
Waited 45 minutes, now I see "Calibration Error 1"...
Not sure what this is or if there is something I need to do?

I see in the code that this is the message reported for a transmitter responding with 0x12 and 0x08, but it is weird because the sensor was otherwise stable before restart.

Any ideas?
Thanks!

**Comment by loopy321** [comment]
I see the error in the [logs](https://gist.github.com/loopy321/7de1f06b7be37f5714025663f9603150#file-gistfile1-txt-L52) , but not clear what happened.

Anyhow, I waited another 30 min and recalibrated, and started getting readings again.  Weird...

**Comment by paulplant** [comment]
Archiving as very old (and we never checked Discussions to see it)
